src/dbup-core/Helpers/IHasher.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DbUp.Helpers
+{
+    /// <summary>
+    /// String hasher interface
+    /// </summary> 
+    public interface IHasher
```

Comment:
The variable names for the class and the class itself are named the same. Can we rename the class? i.e. HashCreator, HashGenerator, HashHelper?

Reply:
Do you mean the class and the interface?

Reply:
Oh, I think I see... you mean the variable in UpgradeConfiguration.cs?

Reply:
I am good if you want to change the property's name in UpgradeConfiguration. My only ask is to pick one, either class & interface or property in UpgradeConfiguration, and change the name.

Reply:
Got it - I renamed `Hasher` to `Sha256Hasher`.